floyd/cli/utils.py

```diff
@@ -32,14 +32,13 @@ def get_module_task_instance_id(task_instances):
 
 def get_data_name(data_str, default=None):
```

Comment:
`default` is no longer used in the function. This can be removed. Just make sure other invocations of this function don't include default.